sass/slider.scss

```diff
@@ -110,6 +115,13 @@ $baseBorderRadius: 4px;
 	@include border-radius($baseBorderRadius);
 }
 
+.slider-track-left, .slider-track-right {
+  position: absolute;
+  background: transparent;
+  .box-sizing(border-box);

```

Comment:
We're using grunt and sass, and this line gives us the following error when including this file:

```
Line 121 of node_modules/bootstrap-slider/sass/slider.scss: Invalid CSS after "  .box-sizing": expected "{", was "(border-box);"
```

Is this valid sass?